GraphQL Local Initiative

benjie opened the pull request.

benjie committed:
GraphQL Local Initiative

benjie committed:
Be clear that a Local is the group not the event

benjie committed:
Announcement blog post

benjie commented on src/pages/community/foundation/local-initiative.mdx:
```diff
@@ -0,0 +1,136 @@
+- **Custom artwork** adhering to the GraphQL Brand Guidelines; including
+  avatars, event banners, and slide templates. \*\*
+
+\* The content of swag bundles may change from time to time. At time of writing
+they are likely to consist of 2-3 t-shirts (sizes to be chosen by organizer),
+stickers, pin-badges, and a GraphQL bow-tie!
```
@jorydotcom what's the latest on this? I wrote it from memory... and I don't have the best memory! :grimacing:

jorydotcom replied:
We will give the meetup organizer a code, and right now the bundle is set up to be 3 t-shirts, 100 stickers, 10 enamel pins and 1 bow tie! I like keeping it a bit more generic now, though, as inventory may change

jorydotcom commented on src/pages/blog/2024-07-22-graphql-local-initiative.mdx:
```diff
@@ -0,0 +1,46 @@
+this, bringing together developers, library maintainers, vendors, and
+enthusiasts to share knowledge and collaborate.
+
+This is why we're pleased to announce the launch of the **GraphQL Local
```
I wonder if we want to say something like, "That's why we've expanded our support for local communities with the launch of the GraphQL Local Initiative" - so we're acknowledging the past efforts of the meetup pro network but also saying, this replaces that program

jorydotcom commented on src/pages/community/foundation/local-initiative.mdx:
```diff
@@ -0,0 +1,136 @@
+- The Local has 1 or more organizer from the area. (We're pretty flexible on the
+  definition of "from the area", just be reasonable.)
+- Each official event will have at least least one **organizer present**.
+- Official events will be held in an appropriate **professional setting**.
```
Should we clarify with some examples? E.g. would a pub qualify, or a games arcade?

benjie replied:
I'm happy to leave this open, if they have questions/venue in mind then they can ask. For example, I think the main room of a pub is probably not an appropriate setting, but exclusive use of the events room in a pub is probably fine.

jorydotcom commented on src/pages/community/foundation/local-initiative.mdx:
```diff
@@ -0,0 +1,136 @@
+  - Schedule
+  - Speaker/talk introduction
+- **Promotion** through GraphQL’s Twitter and LinkedIn channels.
+- **Custom artwork** adhering to the GraphQL Brand Guidelines; including
```
I think we want to be clear that this is not, "we will make custom graphics for each of your meetups" - I think we should just call this Local Artwork so that we don't indicate that we are going to do custom graphic design work

benjie replied:
Good point; have renamed to "Local artwork" throughout.

jorydotcom commented on src/pages/community/foundation/local-initiative.mdx:
```diff
@@ -0,0 +1,136 @@
+  **[GraphQL Brand Guidelines](https://graphql.org/brand/)**.
+- The Local accepts **GraphQL TSC oversight**:
+  - At least one
+    [GraphQL Technical Steering Committee (TSC) member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members)
```
I think we need this to just be you; I am a little concerned this might feel like a big hill if the Local person doesn't know a TSC member and may be afraid to ask someone they don't know to be an admin

benjie replied:
That's a good point, I hadn't considered that they might think they need to ask a TSC member directly. I've added parenthesis to indicate that _we_ will assign a TSC member to make this clearer. I was going to add that they could request a specific TSC member but figured that this is unnecessary - also you're right it'll probably be me for most of them.

jorydotcom commented on src/pages/community/foundation/local-initiative.mdx:
```diff
@@ -0,0 +1,136 @@
+If your gathering is approved, you will be provided with contact information for
+member organizations who have expressed interest in supporting gatherings.
+
+## Apply now!
```
Do we want to say "Apply" or "Sign Up"? We had been talking like this is a "sign up and you get it, but you have to adhere to these to stay on" kind of thing.

benjie replied:
I've modified the language, I don't think "sign up" is quite right either since there is a process albeit a short one. Let me know what you think of the alternative wording.

jorydotcom reviewed (commented):
I have a few comments / thoughts to refine this a little!

benjie committed:
Address Jory's feedback

benjie commented:
Thanks for the input @jorydotcom! I think I've addressed your feedback, and also added another couple minor changes - let me know what you think? (It's all in that last commit.) In particular based on feedback by @jemgillam I wanted to make it explicit that we support both new and existing meets.

bignimbus commented:
LGTM!

benjie committed:
Re-date post

benjie committed:
Fix reference
